src/components/formComponentTemplateTxnOps.tsx

```diff
@@ -237,6 +288,70 @@ export const formComponentTemplateTxnOps = ({
         ),
         validate: validate.publicKey,
       };
+    case "home_domain":
+      return {
+        render: (templ: TemplateRenderProps) => (
+          <TextPicker
+            key={id}
+            id={id}
+            label="Home Domain"
+            placeholder="Ex: example.com"
+            labelSuffix={!templ.isRequired ? "optional" : undefined}
+            value={templ.value || ""}
+            error={templ.error}
+            onChange={templ.onChange}
+            infoLink="https://developers.stellar.org/docs/glossary#home-domain"
+          />
+        ),
+        validate: null,
+      };
+    case "inflation_dest":
```

Comment:
do we need `inflation_dest`? It's deprecated ([doc](https://developers.stellar.org/docs/learn/encyclopedia/inflation))

Reply:
We have it in the current one :woman_shrugging: